x/ethbridge/keeper/keeper_test.go

```diff
@@ -0,0 +1,255 @@
+package keeper
+
+import (
+	"encoding/json"
+	"github.com/Sifchain/sifnode/x/ethbridge/types"
+	"github.com/Sifchain/sifnode/x/oracle"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+var (
+	cosmosReceivers, _                         = CreateTestAddrs(1)
+	amount, _                                  = strconv.ParseInt("10", 10, 64)
```

Comment:
This is very inelegant.  Should just do `amount int64 = 10`

Reply:
sounds good, if curious, the intent was to mimic the input from the cli [here](https://github.com/Sifchain/sifnode/blob/eef610f3be6e361d2b6f6d3a5865d5a3d4634e51/x/ethbridge/client/cli/tx.go#L77)